src/fontem.c

```diff
@@ -349,11 +358,57 @@ static void store_bitmap(FILE *c, FT_Bitmap *bitmap, char *bname, wchar_t ch, in
 			}
```

Comment:
Both rotate and mono should be supported for RLE compressed data; I think this means we should refactor this whole function into three stages; 1 to extract/translate the bitmap, 2 optionally compress it, generating a new bitmap, 3 emitting it as the C sequence.

Reply:
Agreed on the refactoring I will take a look and see if I can spend some time on it. Output and compress might be one step. You add newlines for uncompressed but not for RLE, unless you wanted to add them 8 or so values.

Another option that would be usefully for me is "invert" as in invert the mono. I think with mono and gray scale you can just use the not operator. I am sending the fonts to a B/W mono screen where 0 is on and 1 is off. And looking to just do the displaying with just a little bit shifting and or'ing instead of pixel by pixel.